src/components/stacked-sheet.js

```diff
@@ -0,0 +1,105 @@
+import { LitElement, html, unsafeCSS } from 'lit-element';
+import { StackedSheetStyles } from '../styles/stacked-sheet-styles.js';
+
+class StackedSheet extends LitElement {
+  static get properties() {
+    return {
+      title: { type: String },
+      width: { type: String },
+      opened: { type: Boolean },
+    };
+  }
+
+  static get styles() {
+    return [StackedSheetStyles];
+  }
+
+  constructor() {
+    super();
+    this.width = '100%';
+    this.title = 'New Sheet';
+    this.opened = false;
+    setTimeout(() => {
+      this.opened = true;
+    }, 100);
+  }
+
+  firstUpdated() {
```

Comment:
Can we change the name of the function to `openSheet`?

Reply:
Unfortunately, no. This is a built-in function in LitElement. https://lit-element.polymer-project.org/guide/lifecycle#firstupdated